clients/web/src/templates/widgets/itemList.jade

```diff
@@ -3,9 +3,15 @@ ul.g-item-list
     li.g-item-list-entry
       if checkboxes
         input.g-list-checkbox(type="checkbox", g-item-cid="#{item.cid}")
-      a.g-item-list-link(g-item-cid="#{item.cid}")
+      a.g-item-list-link.g-right-border(g-item-cid="#{item.cid}")
         i.icon-doc-text-inv
         | #{item.get('name')}
+      a(title="Download item" href=item.downloadUrl())
+        i.icon-download
+      if item.collection.length === 1
```

Comment:
Ok, I'll zap that commit out hang on.

roni

On Wed, Jun 29, 2016 at 4:41 PM Michael Fogleman <notifications@github.com>
wrote:

> In clients/web/src/templates/widgets/itemList.jade
> <https://github.com/girder/girder/pull/1407#discussion_r69023294>:
>
> >          i.icon-doc-text-inv
> >          | #{item.get('name')}
> > +      a(title="Download item" href=item.downloadUrl())
> > +        i.icon-download
> > +      if item.collection.length === 1
>
> I don't think you can easily - I say forget about it for now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/girder/girder/pull/1407/files/d8685ccf30b7571ebeb6b879b2fa35f22dd66bac#r69023294>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACxNJOV86xTEn9RxCpOV3uDJ_qDvnlCLks5qQthhgaJpZM4I8MLI>
> .
>